src/content/advanced/etcd/index.md

```diff
@@ -0,0 +1,65 @@
+---
+linkTitle: Etcd Quota Backend Bytes
+title: Etcd Quota Backend Bytes
+description: Since version 18.3 it's now possible to adjust the Etcd `--quota-backend-bytes` which allows to increase Etcd's keyspace database size.
```

Comment:
I'd rather specify also the minor version. was it since 18.3.0?

Reply:
Done. 
Although we don't have any other minor version than 18.3.0 so in reality any 18.3.X version support this.